### PR TITLE
Add acceptance test to clearTrashbin

### PR DESCRIPTION
**Pull request by jasson99**
## Description
Add acceptance test to clear trashbin

## Related Issue

- Fixes issue #2581 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/acceptance/features/webUITrashbin/trashbinDelete.feature [hunk 1]
```diff
@@ -66,3 +66,8 @@ Feature: files and folders can be deleted from the trashbin
     Then file "lorem.txt" should not be listed on the webUI
     When the user reloads the current page of the webUI
     Then file "lorem.txt" should not be listed on the webUI
+
+  Scenario: Clear trashbin
+    When the user clears the trashbin
+    Then the success message "Trash bin was successfully emptied" should be displayed on the webUI
+    And the folder should be empty on the webUI
```
Comment (kiranparajuli589) [inline tests/acceptance/features/webUITrashbin/trashbinDelete.feature]:
```suggestion
    And the trash-bin should be empty on the webUI
```

Comment (jasson99) [reply tests/acceptance/features/webUITrashbin/trashbinDelete.feature]: thanks.done

File: tests/acceptance/stepDefinitions/generalContext.js [hunk 2]
```diff
@@ -26,6 +26,13 @@ Then('the error message {string} should be displayed on the webUI', function (fo
     .expect.element('@message').text.to.equal(folder)
 })
 
+Then('the success message {string} should be displayed on the webUI', function (message) {
```
Comment (dpakach) [inline tests/acceptance/stepDefinitions/generalContext.js]:
```suggestion
Then('the success/error message {string} should be displayed on the webUI', function (message) {
```
And remove the similar method above this.

File: tests/acceptance/stepDefinitions/generalContext.js [hunk 3]
```diff
@@ -26,6 +26,13 @@ Then('the error message {string} should be displayed on the webUI', function (fo
...
+  return client
+    .page.phoenixPage()
+    .waitForElementVisible('@message')
+    .expect.element('@message').text.to.equal(message)
```
Comment (skshetry) [inline tests/acceptance/stepDefinitions/generalContext.js]: Make a function that returns you the message and assert here.

File: tests/acceptance/stepDefinitions/filesContext.js [hunk 4]
```diff
@@ -441,6 +445,11 @@ Then('the folder should be empty on the webUI', async function () {
   return client.assert.equal(allFileRows.value.length, 0)
 })
 
+Then('the trashbin should be empty on the webUI', async function () {
+  const allFileRows = await client.page.FilesPageElement.filesList().allFileRows()
+  return client.assert.equal(allFileRows.value.length, 0)
```
Comment (skshetry) [inline tests/acceptance/stepDefinitions/filesContext.js]:
```suggestion
  return client.assert.strictEqual(allFileRows.value.length, 0)
```

Comment (jasson99) [reply tests/acceptance/stepDefinitions/filesContext.js]: done

**Pull request merged**